packages/ovh-manager-welcome/ovh-manager-welcome.js

```diff
@@ -1,8 +1,9 @@
 import angular from 'angular';
 import uiRouter from '@uirouter/angularjs';
+import ocLazyLoad from 'oclazyload';
 import routing from './ovh-manager-welcome.routes';
 
-const moduleName = angular.module('ovhManagerWelcome', [uiRouter])
+const moduleName = angular.module('ovhManagerWelcome', [ocLazyLoad, uiRouter])
```

Comment:
`const moduleName` sounds useless so you can just use a regular: `export default …` isn't it?